extensions/pass/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "pass",
+  "title": "Pass",
+  "description": "An extension to handle the standard unix password manager in a convenient way using Raycast",
+  "icon": "command-icon.png",
+  "author": "capipo",
+  "categories": [
+    "Security",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Pass",
```

Comment:
```suggestion
      "title": "Search in Pass",
```

Reply:
Thanks for the comments, it's ready to review again.